akka-actor/src/main/scala/akka/actor/Actor.scala

```diff
@@ -293,6 +293,8 @@ final case class UnhandledMessage(
     with AllDeadLetters
 
 /**
+ * Superseeded by [[akka.pattern.ReplyWithStatus]], prefer that when possible.
+ *
```

Comment:
Why not deprecate it?

Reply:
I think that is a bit too invasive on the classic side where it is probably used quite a lot. On the typed side it isn't really supported so it already sort of is deprecated there.